core/modules/rproc/InfileMerger.cc

```diff
@@ -258,6 +247,23 @@ bool InfileMerger::merge(std::shared_ptr<proto::WorkerResponse> response) {
         return true;
     }
 
+    size_t tResultSize;
```

Comment:
Should this be rather called `taskResultSize`? Right now one has to guess what's the meaning of `t`, whether it stands for "temporary", "table", or "total". There are too many variable containing "ResultSize" in this code.